### PR TITLE
CHANGELOG / RELEASE_NOTES for 12.0.6

### PR DESCRIPTION
**Pull request by sdelano**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,8 +7,7 @@
 * No longer generate /etc/opscode/pivotal.cert as it is no longer used.
 * Remove the public key we now use for bootstrapping (/etc/opscode/pivotal.pub) post bootstrap so that it only lives in the database.
 * Disable jmxremote in solr4's Java options
-* Configuration options for the key cache are not exposed to
-/etc/opscode/chef-server.rb
+* Configuration options for the key cache are not exposed to /etc/opscode/chef-server.rb
```
Comment (marcparadise) [inline CHANGELOG.md]: "are now exposed"?

Comment (stevendanna) [inline CHANGELOG.md]: d'oh, good catch.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -2,19 +2,20 @@
...
 
 ### opscode-omnibus
 * Use a cert instead of a public key for the superuser.
 * No longer generate /etc/opscode/pivotal.cert as it is no longer used. Now the public key for the superuser lives in the database and no longer exists on disk.
-* oc\_erchef
-  * Key API Updates - GET, PUT and DELETE  of named keys now supported
-  * Policyfile validation support. (See API Changes, below.)
 
-### Closed Issues
+### oc\_erchef
```
Comment (marcparadise) [inline RELEASE_NOTES.md]: Been trying a new format for these for the last couple, which has received positive reception. You can see it below - it's less of a per-component view (which isn't really of as much interest) as a top-down "what's changed for me" view.

**Comment by stevendanna** [comment]
https://github.com/chef/chef-server/issues/119 
https://github.com/chef/chef-server/issues/97
https://github.com/chef/chef-server/issues/17

I believe these will be fixed for the first time in this release.

**Comment by sdelano** [comment]
Thanks @stevendanna! I went off of the LeanKit board for the issues that we merged in, and there were only those two.

@marcparadise - Can you clarify the per-component thing? `oc_erchef` was a bullet item under `opscode-omnibus` in the previous version, which didn't make much sense at all, hence the reorganization.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -2,19 +2,20 @@
...
+* Key API Updates - GET, PUT and DELETE  of named keys now supported
+* Policyfile validation support. (See API Changes, below.)
 
-*
+### Closed Issues
+* [chef-server-115][https://github.com/chef/chef-server/issues/115]
```
Comment (marcparadise) [inline RELEASE_NOTES.md]: `[name](link)`

**Comment by marcparadise** [comment]
:-1:

**Comment by marcparadise** [comment]
:+1:

File: RELEASE_NOTES.md (hunk 3 above)
Comment (sdelano) [inline RELEASE_NOTES.md]: Yeah, was making this consistent with previous releases, but I like links myself. Will fix up

**Pull request merged**